modules/core/src/v2/coins/abstractEthLikeCoin.ts

```diff
@@ -1,8 +1,12 @@
 /**
  * @prettier
  */
-import * as Bluebird from 'bluebird';
 import { CoinFamily, BaseCoin as StaticsBaseCoin } from '@bitgo/statics';
+import { Eth, getBuilder } from '@bitgo/account-lib';
+import * as Bluebird from 'bluebird';
```

Comment:
Not requesting any changes, but we can use native async/await now for new code.

Reply:
Oh thats awesome! Good to know

Reply:
oh also just realized i'm not even using `co` here :D